source/python/README.md

```diff
@@ -429,7 +431,7 @@ not be used.
 | Type | Attribute | Description |
 | --- | --- | --- |
 | [`EclipseKind`](#EclipseKind) | `kind` | The type of solar eclipse: `EclipseKind.Partial`, `EclipseKind.Annular`, or `EclipseKind.Total`. |
-| [`Time`](#Time) | `peak` | The date and time when the solar eclipse is darkest. This is the instant when the axis of the Moon's shadow cone passes closest to the Earth's center. |
+| [`Time`](#Time) | `peak` | The date and time of the eclipse at its peak. |
```

Comment:
Maybe you forked the code before some significant work was done? Some work was lost here in the comments where I say "eclipse is darkest" instead of "at its peak". I was trying to improve the comments by not defining a work using the same word, LOL.

Reply:
I clone this repo locally yesterday just some moments before you suggested to fix that jitpack issue! Really not sure what is going on! I think it reads it from somewhere else, perhaps from pip?

Reply:
Let me try bringing your pull request to my local system in a separate branch so I can study what is going on better. I will let you know what I find. I do like your idea.